src/qt/languages/de-DE.po

```diff
@@ -1818,6 +1818,27 @@ msgstr "Übertragungsgeschwindigkeit"
 msgid "EMS mode"
 msgstr "EMS-Modus"
 
+msgid "IBM PC/XT 32K Memory Expansion Option"
+msgstr "IBM PC/XT 32K Speichererweiterungsoption"
+
+msgid "IBM PC/XT 64K Memory Expansion Option"
+msgstr "IBM PC/XT 64K Speichererweiterungsoption"
+
+msgid "IBM PC/XT 64/256K Memory Expansion Option"
+msgstr "IBM PC/XT 64/256K Speichererweiterungsoption"
+
+msgid "Generic PC/XT Memory Expansion"
+msgstr "Generische PC/XT Speichererweiterung"
+
+msgid "Microsoft RAMCard for IBM PC"
+msgstr "Microsoft RAMCard für IBM PC"
+
+msgid "IBM PC/AT 128 KB Memory Expansion Option"
+msgstr "IBM PC/AT 128 KB Speichererweiterungsoption"
```

Comment:
But if these are the official names, why is the Voodoo 1 or 2 Graphics also called "Voodoo 1 oder 2 Grafik" in the german translation. even if "Voodoo Graphics" or "Voodoo 1/2" would be correct, because that's the "official name". It feels odd if something that can be translated in a way that makes sense is still in English, even if it's set to German.

Reply:
Look at the front page of IBM's technical references: "Memory Expansion Option".
Now look at the box or manual of a Voodoo card: no "Graphics".

No, the Graphics is not part of the official name, hence why it's translated. A far better example would have been CGA, MDA, PC, etc., which are not translated.